HW12/hw_12_task_3.py

```diff
@@ -0,0 +1,38 @@
+# 3. (необов'язкове виконання) Переписати декоратор із першого завдання, щоб він приймав цілочисельний аргумент `times`.
+# Стільки разів виконувавати друк назви функції і часу, скільки ‘times’ задано.
+
+import time
+
+times = input("Enter your number: ")
+
+def my_decorator(func):
+    # decorator logic
+    called_time = time.strftime("%Y-%m-%d %H:%M:%S", time.localtime())
+    def wraper(*args, **kwargs):
+        try:
+            times.isdigit()
+            print(f"Function '{func.__name__}' was called on {called_time}\n" * int(times))
+            res = func(*args, **kwargs)
+            return res
+        except ValueError:
+            print(f"'{times}' is not a number! Try again!")
+    return wraper
+
+@my_decorator
```

Comment:
times should be set as a parameter for the decorator
```suggestion
@my_decorator(times=2)
```